Scoop/Controller/Ride Posting/TripExtraDetailsViewController.swift

```diff
@@ -18,26 +18,62 @@ class TripExtraDetailsViewController: UIViewController {
     private let peopleImageView = UIImageView()
     
     private let dateLabel = UILabel()
-    private let dateTextField = UITextField()
+    private let datePicker = UIDatePicker()
     private let calendarImageView = UIImageView()
     
     private let timeLabel = UILabel()
-    private let timeTextField = UITextField()
+    private let timePicker = UIDatePicker()
     private let clockImageView = UIImageView()
     
     private let detailsLabel = UILabel()
-    private let detailsTextField = UITextField()
+    private let detailsTextView = UITextView()
     
     private let labelSpace = 5
     private let fieldSpace = 40
-
+    
+    private var ride: Ride!
+    
+    init(ride: Ride) {
+        super.init(nibName: nil, bundle: nil)
+        self.ride = ride
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        
-        navigationItem.title = "Trip Details"
         view.backgroundColor = .white
+        navigationItem.title = "Trip Details"
```

Comment:
consider factoring out strings into a constants file in a future PR. this can include the strings for error messages